pkg/handlers/interface.go

```diff
@@ -33,6 +33,11 @@ type ExperimentAPI interface {
 	ListExperiments(w http.ResponseWriter, r *http.Request)
 }
 
+// NotificationsAPI defines support for notification via server side events
+type NotificationsAPI interface {
```

Comment:
super nit, but other interfaces in this package use the singular tense. (e.g. `NotificationAPI`)